test/integration/acceptance/basic_mst_state_propagation.cpp

```diff
@@ -1,111 +0,0 @@
-/**
```

Comment:
Why is that removed?

Reply:
it is replaced with `test/integration/acceptance/fake_peer_example_test.cpp`